src/Centreon/Application/Controller/MonitoringHostsController.php

```diff
@@ -301,4 +226,41 @@ public function getServicesByHost(int $hostId, RequestParametersInterface $reque
             return View::create(null, Response::HTTP_NOT_FOUND, []);
         }
     }
+
+    /**
+     * Entry point to get all servicegroups attached to host-service
+     *
+     * @IsGranted("ROLE_API_REALTIME", message="You are not authorized to access this resource")
+     * @Rest\Get(
+     *     "/monitoring/hosts/{hostId}/services/{serviceId}/servicegroups",
+     *     condition="request.attributes.get('version.is_beta') == true")
+     *
+     * @param RequestParametersInterface $requestParameters Request parameters used to filter the request
+     * @return View
+     * @throws \Exception
+     */
+    public function getServiceGroupsByHostAndService(
```

Comment:
this should be in MonitoringServicesController ?

Reply:
nope because it is under /monitoring/hosts, I split them by url segment in api request

Reply:
I don't agree, what will happen if tomorrow we decide to change the url, will we have to move the method?
So, always think business and don't rely on the format of the url. This method should be in the MonitoringServicesController.